support-frontend/assets/helpers/abTests/__tests__/abtestTest.js

```diff
@@ -602,10 +602,10 @@ it('targetPage matching', () => {
   const withAcquisitionParams = '/uk/subscribe?INTCMP=header_support_subscribe&acquisitionData=%7B"componentType"%3A"ACQUISITIONS_HEADER"%2C"componentId"%3A"header_support_subscribe"%2C"source"%3A"GUARDIAN_WEB"%2C"referrerPageviewId"%3A"k8heft91k5c3tnnnmwjd"%2C"referrerUrl"%3A"https%3A%2F%2Fwww.theguardian.com%2Fuk"%7D';
   expect(targetPageMatches(withAcquisitionParams, subsShowcaseAndDigiSubPages)).toEqual(true);
   expect(targetPageMatches('/us/subscribe/digital?test=blah', subsShowcaseAndDigiSubPages)).toEqual(true);
- // Test nonGiftLandingAndCheckout regex
- expect(targetPageMatches('/uk/subscribe/digital', nonGiftLandingAndCheckout)).toEqual(true);
- expect(targetPageMatches('/subscribe/digital/checkout', nonGiftLandingAndCheckout)).toEqual(true);
- expect(targetPageMatches('/uk/subscribe/digital/gift', nonGiftLandingAndCheckout)).toEqual(false);
+  // Test nonGiftLandingAndCheckout regex
+  expect(targetPageMatches('/uk/subscribe/digital', nonGiftLandingAndCheckout)).toEqual(true);
+  expect(targetPageMatches('/subscribe/digital/checkout', nonGiftLandingAndCheckout)).toEqual(true);
+  expect(targetPageMatches('/uk/subscribe/digital/gift', nonGiftLandingAndCheckout)).toEqual(false);
```

Comment:
I guess this sort of snuck in here, but the original makes me sad so I'm very happy to keep it!